libethereum/Executive.cpp

```diff
@@ -497,23 +497,24 @@ bool Executive::go(OnOpFunc const& _onOp)
         }
         catch (InternalVMError const& _e)
         {
-            cwarn << "Internal VM Error (" << *boost::get_error_info<errinfo_evmcStatusCode>(_e) << ")\n"
-                  << diagnostic_information(_e);
+            cerror << "Internal VM Error (EVMC status code: "
+                 << *boost::get_error_info<errinfo_evmcStatusCode>(_e) << ")";
             revert();
             throw;
         }
         catch (Exception const& _e)
         {
             // TODO: AUDIT: check that this can never reasonably happen. Consider what to do if it does.
-            cwarn << "Unexpected exception in VM. There may be a bug in this implementation. " << diagnostic_information(_e);
+            cerror << "Unexpected exception in VM. There may be a bug in this implementation. "
+                 << diagnostic_information(_e);
             exit(1);
```

Comment:
That these two exceptions are doing an abort, just reinforces the comment (https://github.com/ethereum/aleth/issues/5212#issuecomment-414805672) reasoning that `InternalVMError` should do the same.

Reply:
These are truly unexpected. Too defensive programming to my taste, I'd let them go to be handled in the main(). But this not the scope of this PR.

Reply:
Well `InternalVMError` is truly unexpected too, isn't it?

Reply:
No it is not, because it can come from external DLLs.
In other cases there is no way to throw `Exception` other than the ones handled above. And `std::exception` can only be `std::bad_alloc`.